.teamcity/src/main/kotlin/util/AdHocPerformanceScenario.kt

```diff
@@ -16,12 +16,12 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
 import jetbrains.buildServer.configs.kotlin.v2019_2.ParameterDisplay
 import jetbrains.buildServer.configs.kotlin.v2019_2.ParametrizedWithType
 
-abstract class AdHocPerformanceScenario(os: Os) : BuildType({
-    val id = "Util_Performance_AdHocPerformanceScenario${os.asName()}"
-    name = "AdHoc Performance Scenario - ${os.asName()}"
+abstract class AdHocPerformanceScenario(os: Os, arch: Arch = Arch.AMD64) : BuildType({
+    val id = "Util_Performance_AdHocPerformanceScenario${os.asName()}${arch.asName()}"
```

Comment:
This changes the old build type id, but I guess it's fine as we don't care too much about the history.